collection/10.17028/rd.lboro.4665448.v1/SIPmetadata/SIP-1.request.json

```diff
@@ -1,7 +1,7 @@
 {
   "messageHeader": {
-    "messageId": "e4bdaea0-4712-4682-8d4b-e6d92b1fe1ac",
     "messageClass": "Command",
+    "messageId": "186369db-99cc-44db-9543-5fdb25a4694c",
```

Comment:
This SIP-1.request.json was tested in jisc-qa environment today and is accepted by the channel-adapter-consumer.